.github/workflows/test.yml

```diff
@@ -49,27 +58,49 @@ jobs:
           dockerfile: dotnet-aspnet/Dockerfile.${{ matrix.ubuntu-release }}
           ignore: DL3008,DL3015,SC3028
 
+      # Build images for multiple archs
       - name: Build the .NET runtime deps container image
         run: |
-          docker build \
-            -t ${{ env.runtime-deps-image-name }} \
+          set -x
+          archs=`cat rockcraft.*.dotnet-deps-6.0-${{ matrix.ubuntu-release }}.yaml | shyaml get-value platforms | shyaml keys`
+          buildx_platforms="linux/$(echo ${archs} | sed 's/ /,linux\//g')"
+
+          docker buildx build \
```

Comment:
Why is buildx necessary? Can't you just run `docker build --platform=<platform> ...`?

Reply:
that is only if `DOCKER_BUILDKIT=1` is set

Reply:
Wouldn't it be simpler to enable buildkit? Then you don't have to install buildx and do the stuff with the tarball output.

Reply:
buildkit is a toolkit to improve the legacy Docker builder. `buildx` is the new and more powerful builder. None of them would remove the need to have an intermediate tarball.